config/locales/user_mailer/en.yml

```diff
@@ -131,8 +131,9 @@ en:
       intro: Personal key used to sign in
       subject: Account Security Alert
     phone_added:
-      help: If you did not make this change, sign in to your profile and manage your
-        phone numbers. We recommend that you also change your password.
+      disavowal_link: here
+      help: If you did not make this change, you can reset your password %{disavowal_link}.
```

Comment:
We may still want to have something here about signing into your profile and managing your numbers.

Reply:
Changed the content to ```If you did not make this change, sign in to your profile and manage your phone numbers. We also recommend that you *reset your password*.``` !